### ISSUE TITLE
Start seeing NullPointerException in 1.5.0

### ISSUE THREAD

**Issue by mars-lan** [software-mansion/react-native-reanimated#551]
```
java.lang.NullPointerException: 
  at com.swmansion.reanimated.nodes.OperatorNode$26.eval (OperatorNode.java:216)
  at com.swmansion.reanimated.nodes.OperatorNode$CompOperator.evaluate (OperatorNode.java:43)
  at com.swmansion.reanimated.nodes.OperatorNode.evaluate (OperatorNode.java:292)
  at com.swmansion.reanimated.nodes.Node.value (Node.java:46)
  at com.swmansion.reanimated.NodesManager.getNodeValue (NodesManager.java:201)
  at com.swmansion.reanimated.nodes.CondNode.evaluate (CondNode.java:22)
  at com.swmansion.reanimated.nodes.Node.value (Node.java:46)
  at com.swmansion.reanimated.nodes.BlockNode.evaluate (BlockNode.java:20)
  at com.swmansion.reanimated.nodes.Node.value (Node.java:46)
  at com.swmansion.reanimated.nodes.FunctionNode.evaluate (FunctionNode.java:18)
  at com.swmansion.reanimated.nodes.Node.value (Node.java:46)
  at com.swmansion.reanimated.nodes.CallFuncNode.evaluate (CallFuncNode.java:44)
  at com.swmansion.reanimated.nodes.Node.value (Node.java:46)
  at com.swmansion.reanimated.nodes.BlockNode.evaluate (BlockNode.java:20)
  at com.swmansion.reanimated.nodes.Node.value (Node.java:46)
  at com.swmansion.reanimated.nodes.AlwaysNode.evaluate (AlwaysNode.java:22)
  at com.swmansion.reanimated.nodes.AlwaysNode.evaluate (AlwaysNode.java:7)
  at com.swmansion.reanimated.nodes.Node.value (Node.java:46)
  at com.swmansion.reanimated.nodes.AlwaysNode.update (AlwaysNode.java:17)
  at com.swmansion.reanimated.nodes.Node.runUpdates (Node.java:130)
  at com.swmansion.reanimated.NodesManager.onAnimationFrame (NodesManager.java:160)
  at com.swmansion.reanimated.NodesManager.access$000 (NodesManager.java:55)
  at com.swmansion.reanimated.NodesManager$1.doFrameGuarded (NodesManager.java:108)
  at com.facebook.react.uimanager.GuardedFrameCallback.doFrame (GuardedFrameCallback.java:28)
  at com.facebook.react.modules.core.ReactChoreographer$ReactChoreographerDispatcher.doFrame (ReactChoreographer.java:174)
  at com.facebook.react.modules.core.ChoreographerCompat$FrameCallback$1.doFrame (ChoreographerCompat.java:84)
  at android.view.Choreographer$CallbackRecord.run (Choreographer.java:994)
  at android.view.Choreographer.doCallbacks (Choreographer.java:794)
  at android.view.Choreographer.doFrame (Choreographer.java:725)
  at android.view.Choreographer$FrameDisplayEventReceiver.run (Choreographer.java:981)
  at android.os.Handler.handleCallback (Handler.java:883)
  at android.os.Handler.dispatchMessage (Handler.java:100)
  at android.os.Looper.loop (Looper.java:237)
  at android.app.ActivityThread.main (ActivityThread.java:7762)
  at java.lang.reflect.Method.invoke (Native Method)
  at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run (RuntimeInit.java:493)
  at com.android.internal.os.ZygoteInit.main (ZygoteInit.java:1047)
```

**Comment by osdnk** [comment]
Can you share some more context please?

**Comment by osdnk** [comment]
Fixes in 1.6.0
